src/smriprep/cli/run.py

```diff
@@ -97,6 +109,13 @@ def get_parser():
         help='a space delimited list of participant identifiers or a single '
         'identifier (the sub- prefix can be removed)',
     )
+    g_bids.add_argument(
+        '--session-id',
```

Comment:
Seems worth being consistent with `--participant-label`.

```suggestion
        '--session-label',
```